src/Runtime/Core/Collections/ImmutableArray.cs

```diff
@@ -2,6 +2,11 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.ComponentModel;
+using System.Data.Common;
+using System.Linq;
+using System.Runtime.CompilerServices;
```

Comment:
Only Linq is needed

Reply:
Sorry. IDE is very helpful at inserting this stuff automatically and I do not even check these. Fixed